packages/input-datepicker/src/LionInputDatepicker.js

```diff
@@ -12,6 +12,7 @@ import { LionCalendarOverlayFrame } from './LionCalendarOverlayFrame.js';
 
 /**
  * @typedef {import('@lion/core').RenderOptions} RenderOptions
+ * @typedef {import('@lion/form-core/types/FormControlMixinTypes.js').ModelValueEventDetails} ModelValueEventDetails
```

Comment:
Nice. Only thing is that this line could be lef out now

Reply:
done